releases/release-1.31/release-notes/maps/pr-123822-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 123822
+releasenote:
+  text: Fixed a missing `status.` prefix on custom resource validation error messages.
```

Comment:
```suggestion
  text: Fixed a missing status. prefix in custom resource validation error messages.
```

Reply:
We might want to remove the period in `status.` here as well?